pages/pipelines/group_step.md.erb

```diff
@@ -0,0 +1,208 @@
+# Group Step
+ 
+A *group* step can contain various sub-steps, and display them in a single logical group on the Build page. For example, you can group all of your linting steps or all of your UI test steps to keep the Build page less messy. Sub-groups and nested groups are not supported.
+
+The group step also helps manage dependencies between a collection of steps, for example, "step X" [`depends-on`](/docs/pipelines/group-step#group-step-attributes) everything in "group Y".
+
+{:toc}
+ 
+A group step can be defined in your pipeline settings or your [pipeline.yml](/docs/pipelines/defining-steps) file.  
+
+Here is an example of using the group step:  
+
+```yml
+- group: "\:broom\: Linting"
+  key: "linting"
+  depends_on: docker
+  steps:
+    - name: "\:rubocop\: RuboCop"
+      command: ".buildkite/steps/rubocop"
+    - name: "\:eslint\: ESlint"
+      command: ".buildkite/steps/eslint"
+    - name: "\:flowtype\: Flow"
+      command: ".buildkite/steps/flow"
+```
+{: codeblock-file="group-step-in-pipeline.yml"} 
```

Comment:
I think it makes sense to call all of these `pipeline.yml` and not custom names per description